### ISSUE TITLE
Pagination current page is missing `aria-current=page` attribute

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-react-bootstrap#78]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-react-bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have validated any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-react-bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

REF: https://modus-react-bootstrap.trimble.com/components/pagination/#Example

![image](https://user-images.githubusercontent.com/1212885/175493691-f3635d85-d198-47a8-9d9f-e80340365958.png)

The current, active page has the following generated HTML

```html
<li class="page-item active">
<span class="page-link">5<span class="sr-only">(current)</span></span>
</li>
```
but it should be:

```html
<li class="page-item active" aria-current="page">
<span class="page-link">5<span class="sr-only">(current)</span></span>
</li>

```

### Reduced test cases

n/a

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

### What version of Modus React Bootstrap are you using?

1.1.0

### Priority

Medium
